data/config-schema/src/main/resources/net/corda/schema/configuration/db/1.0/corda.db.json

```diff
@@ -64,6 +64,30 @@
                   "type": "null"
                 }
               ]
+            },
+            "idleTimeout": {
+              "description": "The maximum time (in minutes) a connection can stay idle in the pool. A value of 0 means that idle connections are never removed from the pool.",
+              "type": "integer",
+              "minimum": 0,
```

Comment:
We have a mixture of minutes and seconds here which strikes me as a bad idea. I expect this comes from the Hikari api? But in any case I think we should standardise on seconds.

Reply:
Yeah all these defaults and time scales were taken from `DataSourceFactory.create`. I do agree they should be standardized.

Reply:
Changed them to be all seconds